src/libultra/gu/sinf.c

```diff
@@ -14,6 +14,10 @@ static const du pilo = { 0x3E6110B4, 0x611A6263 };
 
 static const fu zero = { 0x00000000 };
 
+/**
+ * @param x radians
```

Comment:
For some the arg is `x` while others it is `angle` (at first I thought it was `x` if it was radians and `angle` for binary angles, but that is also not the case). Feel like these could/should be standardized.

Reply:
Thoughts on `Math_CosF(f32 rad)`, `Math_CosS(s16 binang)` then? sounds like a good idea

Reply:
Fine by me.